doc/caveats.md

```diff
@@ -0,0 +1,41 @@
+Caveats
+=======
+
+Git-dit is, obviously, git specific and will hence not work with other VCS like
+mercurial. Import and export functionality for other issue tracking systems are,
+however, not inconceivable. Functionality like referencing issues across
+repositories will, however, most likely never be provided across different VCS.
```

Comment:
Wouldn't include this sentence. Do not tell people what won't be there; just tell them what _will_ be there.